src/mdesktopentry.cpp

```diff
@@ -347,6 +375,41 @@ QString MDesktopEntry::value(const QString &group, const QString &key) const
     return d_ptr->keyFile.stringValue(group, key);
 }
 
+QString MDesktopEntry::localizedValue(const QString &key) const
+{
+    QStringList v = key.split('/');
+    return (v.length() == 2) ? localizedValue(v[0], v[1]) : QString();
```

Comment:
Would it break something if I changed also `value()` to behave that way too? I don't think it would but just wanted to check if you think it's safe. I'm just thinking that it would be nice to have symmetry between these two versions (localized and non-localized). If `value()` can't be change, maybe it would be best to leave this out for least confusion.

Reply:
Oh right, was myself only checking the new methods. Symmetry is good indeed.

I think value() should be safe to change, can break something only if it was passing an invalid key and expecting to get an empty string back.

Reply:
Changed it to default to `DesktopEntrySection`.